### ISSUE TITLE
[TypeScript]Unable to import names

### ISSUE THREAD

**Issue by yutak23** [softonic/axios-retry#251]
We cannot use `name import`.

```
srv/app.ts:2:22 - error TS2614: Module '"axios-retry"' has no exported member 'IAxiosRetryConfig'. Did you mean to use 'import IAxiosRetryConfig from "axios-retry"' instead?

2 import axiosRetry, { IAxiosRetryConfig } from 'axios-retry';
                       ~~~~~~~~~~~~~~~~~
```

I noticed this when @rchl pointed it out to me. My apologies.
https://github.com/softonic/axios-retry/pull/250#issuecomment-1810403564

**Comment by borisilic-ap** [comment]
Similar issue here where `exponentialDelay` can no longer be de-structured like so

```
// TS2614: Module  'axios-retry'  has no exported member  exponentialDelay . 
import axiosRetry, { exponentialDelay } from 'axios-retry'
```

This happens with the change from `3.8.2` to `3.9.0`. 

This is a breaking change, the version should be bumped to `4.0.0`.

**Comment by mindhells** [comment]
closed as #250 has been reverted on 3.9.0
The plan is now to migrate to TS implementation
Thanks for your feedback!
